src/frontend/App/context/toggles.ts

```diff
@@ -8,6 +8,7 @@ export enum ToggleName {
     opprettBehandlingForFerdigstiltJournalpost = 'familie.ef.sak.opprett-behandling-for-ferdigstilt-journalpost',
     kanMigrereBarnetilsyn = 'familie.ef.sak.migrering.barnetilsyn',
     papirsoknadTerminbarnRevurdering = 'familie.ef.sak.papirsoknad-og-terminbarn-paa-revurdering',
+    henleggBehandlingUtenÅHenleggeOppgave = 'familie.ef.sak.henlegg-behandling-uten-oppgave',
```

Comment:
Ikke superviktig, men skulle kanskje vært 'familie.ef.sak.**frontend**.henlegg-behandling-uten-oppgave'?

Reply:
Kanskje, men den brukes to steder i backend også 🤔